husky_components/overlay/main.js

```diff
@@ -68,19 +62,14 @@ define([], function() {
     var defaults = {
             trigger: 'click',
             triggerEl: null,
-            verticalSpacing: 100, //px
             instanceName: 'undefined',
```

Comment:
@marcelmoosbrugger i think '' would be better than 'undefined' as it fits our usual strategy